com.obsproject.Studio.Plugin.obs_kinect.yaml

```diff
@@ -0,0 +1,113 @@
+id: com.obsproject.Studio.Plugin.obs_kinect
```

Comment:
Unless you're already bound to this id, I'd suggest using `com.obsproject.Studio.Plugin.Kinect` to match what other plugins are doing (if you agree, please also update all paths containing `/app/plugins/obs_kinect` too)

Reply:
This works fine. Tbh, I just copied the plugin name and replaced the dash with underscore to follow freedesktop naming. I should've asked this when I did it the first time, but do I have to close this PR to change the branch name or will you guys take care of that?